### DISCUSSION TITLE
如何避免轮播组件切换箭头点击后触发form表单必填校验

### DISCUSSION THREAD

**Discussion by ctrl-alt-xxh** [layui/layui#2052] [建议]
项目中使用了轮播组件发现一个情况，由于轮播组件是放在form表单内，点击切换箭头时，触发了表单项必填校验

检查页面元素后，发现切换箭头采用button实现，而button没有设置type="button"属性（默认是submit）
请问各位同学是否有方法避免触发默认的submit行为？或者社区是否可以将这些button元素明确指定type属性呢？

**Comment by sentsim** [comment]
确实存在这个影响，需要添加 `type="button"`
已标注 Issue #2053
